tests/unit/moduleapi/dryrun.tcl

```diff
@@ -0,0 +1,36 @@
+set testmodule [file normalize tests/modules/dryrun.so]
+
+start_server {tags {"modules"}} {
+    r module load $testmodule
+
+    test {test Dry Run - OK OOM} {
+        set val [r get x]
```

Comment:
maybe i'm missing something, but who sets the initial value of `x`?
seems like this line will throw an exception.
instead of getting the previous value, i suggest to just set one, and then validate that it survived.

Reply:
ok.